mllib/src/main/scala/org/apache/spark/ml/tree/impl/RandomForest.scala

```diff
@@ -740,7 +740,7 @@ private[ml] object RandomForest extends Logging {
               val categoryStats =
                 binAggregates.getImpurityCalculator(nodeFeatureOffset, featureValue)
               val centroid = if (categoryStats.count != 0) {
-                categoryStats.predict
+                categoryStats.prob(categoryStats.predict)
```

Comment:
As I saw from the implementation, `categoryStats.stats(1)` is just the count of class 1. Are we going to order bins by that?